service/tracker/execution.go

```diff
@@ -159,7 +171,40 @@ func (e *Execution) processNext() error {
 	// Dump the block execution record into our cache and push all trie updates
 	// into our update queue.
 	e.records[blockID] = record
+	if e.execClient != nil {
+		e.log.Debug().Hex("block_id", blockID[:]).Msg("fetching updates from data sync")
+		e.log.Debug().Int("updates", len(record.TrieUpdates)).Msg("got trie updates from GCP")
+		// get Trie updates from exec data sync
+		req := &access.GetExecutionDataByBlockIDRequest{BlockId: blockID[:]}
+		res, err := e.execClient.GetExecutionDataByBlockID(context.Background(), req)
+		if err != nil {
+			return fmt.Errorf("could not get execution data from access node: %w", err)
+		}
 
+		execTrieUpdates := make(map[string]bool, 0)
```

Comment:
better to extract the logic of comparing two sets of trie updates into a separate function.